test/unit/log.provider.test.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback-next-extension-starter
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {sinon} from '@loopback/testlab';
+import {ParsedRequest} from '@loopback/core';
+import {LogProvider} from '../..';
+
+describe('LogProvider', () => {
+  // tslint:disable-next-line:no-any
+  let spy: any;
+  beforeEach(() => {
+    spy = sinon.spy(console, 'log');
+  });
+
+  afterEach(() => {
+    spy.restore();
+  });
+
+  it('logs a value without a start time', () => {
+    const match = '/test : () => test message';
```

Comment:
Please add `^` and `$` to verify the whole contents of the message.

Reply:
I tried converting `match` to a RegExp as follows: 
`const match = new RegExp('^\/time : \(\) => test message$');`

but it keeps resulting in the following error:
```
 1) LogProvider (unit) logs a value without a start time:
     AssertError: expected log to be called with arguments 
/test : () => test message match(/^\/time : () => test message$/)
      at Object.fail (node_modules/sinon/lib/sinon/assert.js:96:21)
      at failAssertion (node_modules/sinon/lib/sinon/assert.js:55:16)
      at Object.assert.(anonymous function) [as calledWith] (node_modules/sinon/lib/sinon/assert.js:80:13)
      at Context.it (dist/test/unit/providers/log-action.provider.unit.js:25:32)
```

so I'm leaving it as a string match for now. `sinon.match(string)` expect the entire expectation to be present as a whole so I think it's fine anyways.

If you can guide me with the correct `RegExp` I can update it.

Reply:
This seems to work for me well:

```ts
 it('logs a value without a start time', () => {
    const match = /^\/test : \(\) => test message/;

    log(req, [], 'test message');
    sinon.assert.calledWith(spy, sinon.match(match));
  });
```

The trick is to wrap the RegExp via `sinon.match()`.